ibis/expr/types/arrays.py

```diff
@@ -5,14 +5,16 @@
 
 from public import public
 
+import ibis
 import ibis.expr.operations as ops
+import ibis.expr.types as ir
 from ibis.common.deferred import Deferred, deferrable
 from ibis.expr.types.generic import Column, Scalar, Value
 
 if TYPE_CHECKING:
     from collections.abc import Iterable
 
-    import ibis.expr.types as ir
+    from ibis.expr.types import dt
```

Comment:
```suggestion
    import ibis.expr.datatypes as dt
```

Definitely do not import internal module aliases from other modules.

Reply:
oops, must have been a blind vscode/copilot acceptance.